.github/workflows/wasm.yml

```diff
@@ -0,0 +1,55 @@
+name: WASM
+
+on:
+  push:
+    branches: ["main"]
+  pull_request:
+    branches: ["main"]
+
+jobs:
+  build:
+    name: Build target ${{ matrix.target }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        target:
+          - wasm32-wasi
```

Comment:
I've just always tested `wasm32-wasi` because it has various improvements and fixes relative to the `wasm32-unknown-unknown` target. But I've gone ahead and added the latter to the job as well, it's easy enough and also appears to pass.

Reply:
Thank you very much!